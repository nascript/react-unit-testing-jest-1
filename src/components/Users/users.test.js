import React from "react";
import { render } from "@testing-library/react";
import { Users } from "./";

describe("<Users />", () => {
  const users = [
    { id: 123, firstName: "Mo", lastName: "Dwina", email: "b@b.com" },
  ];
  test("renders component", () => {
    const { container } = render(<Users users={users} />);
    expect(container).toMatchSnapshot();
  });
});
