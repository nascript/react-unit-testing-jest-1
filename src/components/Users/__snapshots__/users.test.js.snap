// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Users /> renders component 1`] = `
<div>
  <div
    class="users"
  >
    <h1>
      Registered users
    </h1>
    <table
      width="100%"
    >
      <thead>
        <tr>
          <th>
            First name
          </th>
          <th>
            Last name
          </th>
          <th>
            Email
          </th>
        </tr>
      </thead>
      <tbody>
        <tr>
          <td>
            Mo
          </td>
          <td>
            Dwina
          </td>
          <td>
            b@b.com
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
